flume-ng-doc/sphinx/FlumeUserGuide.rst

```diff
@@ -1401,6 +1402,7 @@ Example for agent named a1:
   a1.sources.r1.headers.f2.headerKey1 = value2
   a1.sources.r1.headers.f2.headerKey2 = value2-2
   a1.sources.r1.fileHeader = true
+  a1.sources.ri.numThreshold = 1000
```

Comment:
the example is different than the parameter name

Reply:
Thanks for catching it. Fixed.